InternetStoreCHIReloaded/DAL/Migrations/20210812141009_AddedManyToManyBetweenCartEntityAndProductWithEntity.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace DAL.Migrations
+{
+    public partial class AddedManyToManyBetweenCartEntityAndProductWithEntity : Migration
```

Comment:
Why relationship between Cart and Cart items is many-to-many?

Reply:
done